oqs-template/generate.yml

```diff
@@ -305,6 +321,7 @@ kems:
     nid_hybrid: '65041'
     oqs_alg: 'OQS_KEM_alg_bike_l1'
     enable_kem: true
+    enable_tls: true
```

Comment:
What about the idea to disable any one KEM (maybe this, maybe another one) to ensure the "disablement logic" flow gets tested?